### ISSUE TITLE
Webhook#build_entity fails when a sales invoice is deleted

### ISSUE THREAD

**Issue by rjdegelder** [maartenvanvliet/moneybird#45]
When a (draft) SalesInvoice is deleted in Moneybird the webhook is called with:

```
{
  "action" => "sales_invoice_updated",
  "administration_id" => "...",
  "controller" => "callbacks/moneybird",
  "entity" => nil,
  "entity_id" => "...",
  "entity_type" => "SalesInvoice",
  "moneybird" => {
    "action" => "sales_invoice_destroyed",
    "administration_id" => "...",
    "entity" => nil,
    "entity_id" => "...",
    "entity_type" => "SalesInvoice",
    "state" => "draft",
    "webhook_id" => "...",
    "webhook_token" => "..."
  },
  "state" => "draft",
  "token" => "...",
  "webhook_id" => "...",
  "webhook_token" => "..."
}
```

This fails in Webhook#build_entity with `undefined method [] for nil:NilClass` because entity is nil (instead of an object with a client attribute as expected in `moneybird/resource:19`).

Backtrace:
```
/GEM_ROOT/gems/moneybird-0.9.8/lib/moneybird/resource.rb:19→ attributes=
/GEM_ROOT/gems/moneybird-0.9.8/lib/moneybird/resource.rb:6→ initialize
/GEM_ROOT/gems/moneybird-0.9.8/lib/moneybird/webhook.rb:17→ new
/GEM_ROOT/gems/moneybird-0.9.8/lib/moneybird/webhook.rb:17→ build_entity
```

**Comment by maartenvanvliet** [comment]
Great find!

Could you file a PR of a failing test or a fix?

**Comment by benjaminwols** [comment]
This Issue can be closed since it is fixed by #61
